components/service/glean/src/test/java/mozilla/components/service/glean/storages/EventsStorageEngineTest.kt

```diff
@@ -0,0 +1,137 @@
+/* Any copyright is dedicated to the Public Domain.
```

Comment:
The license here is odd .. I think we prefer this to be MPL2?

Reply:
Hi @st3fan ! I come from the Firefox-Desktop world, in which test files are usually licensed as public domain, as per our [Mozilla Policy](https://www.mozilla.org/en-US/MPL/headers/). I thought that also applied on this repo, since I saw at least [another usage](https://github.com/mozilla-mobile/android-components/blob/210751ad2b02639cbfc729b0e96a0c2dc204e846/components/support/utils/src/test/java/mozilla/components/support/utils/WebURLFinderTest.java). But looking at other files, I found that some tests are licensed as MPL as well.